build.gradle

```diff
@@ -15,16 +15,17 @@ repositories {
 
 dependencies {
     compile group: 'com.googlecode.json-simple', name: 'json-simple', version: '1.1.1'
-    compile group: 'io.springfox', name: 'springfox-swagger-ui', version: '2.6.1'
-    compile group: 'io.springfox', name: 'springfox-swagger2', version: '2.6.1'
+    compile group: 'io.springfox', name: 'springfox-swagger-ui', version: '2.9.2'
+    compile group: 'io.springfox', name: 'springfox-swagger2', version: '2.9.2'
+    compile group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa'
+    compile group: 'mysql', name: 'mysql-connector-java', version: '8.0.16'
+    testCompile 'org.junit.jupiter:junit-jupiter-api:5.6.0'
     implementation 'org.springframework.boot:spring-boot-dependencies:2.2.5.RELEASE'
```

Comment:
I have removed this dependency `testCompile 'org.junit.jupiter: junit-jupiter-api: 5.6.0'` before because there were problems running unit tests.
Did you fix the problems and tried the unit tests by adding this library?